docs/admin/access_control/service_accounts.mdx

```diff
@@ -0,0 +1,50 @@
+
+## Creating Service Accounts
+
+Service accounts are created like regular user accounts, but with a few key differences.
+
+- Go to **Site admin** → **Users & auth** → **Users**
+- Click **Create User**
+- Enter a descriptive **Username** (e.g., `ci-automation`, `api-integration`)
+- Check the **Service account** checkbox
+- Click **Create service account**
+
+You'll be presented with some next steps you might want to take, like creating an access token, managing and assigning roles, and managing repository permissions.
+
+- Service accounts are automatically assigned the "Service Account" system role
+- They appear in the user list with "Service account" type designation
+
+## Managing Access Tokens
+
+Service accounts authenticate using access tokens rather than passwords. For detailed information about creating, managing, and using access tokens, see:
+
+- [Creating an access token](/cli/how-tos/creating_an_access_token)
+- [Managing access tokens](/cli/how-tos/managing_access_tokens)
+- [Revoking an access token](/cli/how-tos/revoking_an_access_token)
+
+Use service account access tokens to access Sourcegraph's [GraphQL API](/api/graphql).
+
+## Role-Based Access Control (RBAC)
+
+Service accounts integrate with Sourcegraph's [role-based access control](/admin/access_control) to provide fine-grained permission control.
+
+### System Roles
+
+Service accounts are automatically assigned the **Service Account** system role, which provides basic API access permissions and standard search capabilities. The **Service Account** system role is applied to all service accounts and can be used to provide service accounts with a default set of permissions. For more specialized service accounts, it is recommended to create custom roles and assign them to service accounts as needed.
+
+### Managing Roles
+
+Administrators can assign additional roles to service accounts through the user management interface. For detailed information on managing roles and permissions, see:
+
+- [Managing roles and permissions](/admin/access_control#managing-roles-and-permissions)
+- [Managing user roles](/admin/access_control#managing-user-roles)
+- [Creating custom roles](/admin/access_control#creating-a-new-role-and-assigning-it-permissions)
+
+## Repository Permissions
+
+Service accounts respect repository permissions and access controls. For comprehensive information about repository permissions, see the [Repository permissions](/admin/permissions) documentation.
+
+Service accounts cannot have external service accounts. If service accounts need access to private repositories, they must explicitly be granted access to those repositories. This can be done from the service account's user settings page, under the **Repo permissions** tab, or via [the GraphQL API](/admin/permissions/api#explicit-permissions-api).
```

Comment:
> Service accounts cannot have external service accounts

as someone who doesn't know our database schema.. i'm not sure I would know what external service accounts are and why not having them is a problem. 

Maybe we can rephrase this a bit to something like 

```
Service accounts by default can only access public and unrestricted repositories in Sourcegraph. You may explicitly grant fine-grained access to private repositories from the service account's user settings page, under the **Repo permissions** tab, or via [the GraphQL API](/admin/permissions/api#explicit-permissions-api).
```